resources/references/core-reference/commands-reference.md

```diff
@@ -251,10 +251,12 @@ $ bin/console [command] [parameters]
 
 ### Scheduled-task
 
-| Command | Description |
+| Command | Description | Version
```

Comment:
@sushmangupta @shyim what do you think about adding a version to the command references in the future?

Reply:
sounds good